app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command_line: str) -> None:
+    (command,
+     file_name,
+     new_file_name) = command_line.split(" ")
```

Comment:
`.split()` is splitting spaces by default, so we don't need to pass any `args` to it in this case